metric/metric.go

```diff
@@ -0,0 +1,58 @@
+package metric
+
+import (
+	"strconv"
+
+	"github.com/prometheus/client_golang/prometheus"
+	log "github.com/sirupsen/logrus"
+)
+
+var logger = log.WithFields(log.Fields{"component": "metrics"})
+
+var (
+	namespace = ""
+	subsystem = "service"
+)
+
+var (
+	reqLabels   = []string{"service", "operation", "code"}
+	reqDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: namespace,
+		Subsystem: subsystem,
+		Name:      "idler_request_duration_seconds",
```

Comment:
this metrics FQ name would be service_idler_request_duration_seconds.  If 'idler' is your service name, better to have 'idler_request_duration_seconds' as FQ name.

Reply:
Great thanks

Reply:
Sorry, now I am a bit confused, you mean I should rename to something else, or leave it as is?

Reply:
I mean the current name of this metric will be 'service_idler_request_duration_seconds'.  Just wanted to take your attention on this.  You can better decide about naming, so it's totally upto you.